src/AssetManager/Controller/ConsoleControllerFactory.php

```diff
@@ -2,18 +2,34 @@
 
 namespace AssetManager\Controller;
 
+use AssetManager\Service\AssetManager;
+use Interop\Container\ContainerInterface;
+use Zend\ServiceManager\AbstractPluginManager;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class ConsoleControllerFactory implements FactoryInterface
 {
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
```

Comment:
If $requestedName and $options aren't being used, shouldn't they be removed?  These are not required for the invoke signature.